---
 Model/HeaderProvider/Hsts.php | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Model/HeaderProvider/Hsts.php b/Model/HeaderProvider/Hsts.php
index b22775e..291e4f8 100644
--- a/Model/HeaderProvider/Hsts.php
+++ b/Model/HeaderProvider/Hsts.php
@@ -25,7 +25,7 @@ class Hsts extends \Magento\Framework\App\Response\HeaderProvider\AbstractHeader
      *
      * @var string
      */
-    protected $headerValue = 'max-age=31536000';
+    protected $headerValue = 'max-age=31536000; includeSubDomains';
 
     /**
      * @var \Magento\Framework\App\Config\ScopeConfigInterface
-- 
2.35.1
